src/main/resources/run-class.sh

```diff
@@ -98,9 +98,16 @@ if [ -e conf/env.conf ]; then
   CONF_FILE="-Dconfig.file=conf/env.conf"
 fi
 
+#Add a cache-key config, user tests for existence
+ADD_CACHE_KEY=""
+if [ -e conf/cacheKey.Sha1 ]; then
+  CACHEKEY = $(<cacheKey.Sha1)
```

Comment:
I don't think this assignment will work with spaces. I get the feeling you're not testing this . . .